helm/image-viewer/values.yaml

```diff
@@ -0,0 +1,61 @@
+# Default values for image-viewer.
+# This is a YAML-formatted file.
+# Declare variables to be passed into your templates.
+
+replicaCount: 1
+serviceName: image-viewer-service
+
+image:
+  repository: gen3-image-viewer
```

Comment:
This is erroring out for me. I assume this is overwritten by the image-viewer repo specified in the secrets file?

Reply:
Exactly.  The stanza I have in my values.yaml:

```
image-viewer:
  serviceName: image-viewer-service
  image:
    repository: quay.io/ohsu-comp-bio/gen3-image-viewer
    pullPolicy: Always
    tag: "latest"
```